### ISSUE TITLE
downloading issue

### ISSUE THREAD

**Issue by faraday08144** [OpenBoard-org/OpenBoard#1056]
i cant download openboard to my windows.am using window10 pls help me

**Comment by letsfindaway** [comment]
What exactly is your problem? The download is available from [the bottom of the release page](https://github.com/OpenBoard-org/OpenBoard/releases/tag/v1.7.1) or [from the OpenBoard web page](https://openboard.ch/download.en.html).

**Comment by faraday08144** [comment]
Yes I will click the button but it will not load the download page it will
be write can't reach the page

**Comment by letsfindaway** [comment]
For me download easily works (on a Linux computer). Here again the full download link: https://github.com/OpenBoard-org/OpenBoard/releases/download/v1.7.1/OpenBoard_Installer_1.7.1.exe

But there have been people complaining that their virus scanner was blocking the download. See #884, #885, #889. But this is not true. The download is free of viruses.

**Comment by faraday08144** [comment]
Am using window 10 sir

**Comment by letsfindaway** [comment]
> Am using window 10 sir

I know, you wrote it, I read it. But did you actually read my previous comment? And did you read the provided links to the other issues discussing this? 

Here again in simple steps:

- You're using Windows. You have problems downloading the installer.
- I'm using Linux, I can easily download even the Windows installer.
- Therefore it is not a server issue but a local issue on your side.
- There have been reported issues that on Windows a virus scanner might consider the installer "dangerous" and blocks it.
- The maintainer of the project promises that there is no virus in the installer and that this might be a false positive.

So what can you do: probably disabling the virus scanner for the download and installation of OpenBoard might help. But as I'm not using Windows I cannot give you more advice.
